src/app/app.component.html

```diff
@@ -1,39 +1,19 @@
-<div
-  style="
-    display: grid;
-    grid-template-columns: 19% 81%;
-    height: 100vh;
-    width: 100vw;
-  "
->
-  <div style="background-color: #21252e">
-    <app-menu
-      class="menu"
-      [menuItems]="items"
-      style="
-        display: flex;
-        flex-direction: column;
-        position: sticky;
-        top: 0;
-        bottom: 0;
-      "
-    >
-    </app-menu>
-  </div>
-  <div style="display: flex; flex-direction: column; position: relative">
-    <app-header></app-header>
-    <router-outlet class="content"></router-outlet>
-    <app-footer
-      class="footer"
-      style="
-        background-color: #1c1d26;
-        height: max-content;
-        position: absolute;
-        bottom: 0;
-        left: 0;
-        right: 0;
-      "
-      >Footer</app-footer
-    >
+<div class="wholeScreen">
+  <div fxLayout="row" style="flex: 1">
```

Comment:
Could we remove "style" attribute from HTML elements and define all the styles in SCSS only?